eval/src/vespa/eval/eval/cell_type.h

```diff
@@ -6,15 +6,19 @@
 #include <vector>
 #include <cstdint>
 #include <cassert>
+#include "int8float.h"
+#include <vespa/vespalib/util/bfloat16.h>
 
 namespace vespalib::eval {
 
-enum class CellType : char { FLOAT, DOUBLE };
+enum class CellType : char { FLOAT, DOUBLE, BFLOAT16, INT8 };
```

Comment:
I am leaning towards switching the ordering of FLOAT and DOUBLE, to make everything have the types in the same order (This would not mean that enum value is tied to the binary format cell type id, even if it happens to be the same).